docs/plus/index.md

```diff
@@ -270,6 +270,7 @@ app.synth();
 
     ```go
     app := cdk8s.NewApp(nil)
+    id := "ubuntu"
     chart := cdk8s.NewChart(app, jsii.String(id), nil)
```

Comment:
Wouldn't it make more sense to just use a string literal instead of assigning to a variable (get rid of `id` altogether)?

Reply:
Sounds good. I have updated the doc to use the literal directly